src/UiArea/DrawMatrix.cc

```diff
@@ -6,6 +6,10 @@ UiDrawMatrix::UiDrawMatrix() {
 	m = new uiDrawMatrix();
 }
 
+void UiDrawMatrix::free() {
```

Comment:
Samer as above, except you didn't  expose this to Js side.
But I think it can be moved to a destructor too...